session2/Workflow.ipynb

```diff
@@ -58,28 +58,41 @@
    },
    "source": [
     "When you run the file this way you tell it to use the current active Python to run the file. \n",
-    "But you can also specify a specific python interpreter in the file itself by adding the following line at the beginning of your file:\n",
+    "\n",
+    "To make this script run like a command, add the following to the beginning of the file:\n",
     "\n",
     "```\n",
-    "#!/Users/<yourname>/anaconda3/bin/python\n",
+    "#!/usr/bin/env python\n",
     "```\n",
     "\n",
-    "Then you can still do:\n",
+    "Then, make the script executable using the command:\n",
     "\n",
     "```\n",
-    "python hello.py\n",
+    "$ chmod u+x hello.py\n",
     "```\n",
     "\n",
-    "But you can also make your file executable and run it alone:\n",
+    "Now you will be able to run the script as:\n",
     "\n",
     "```\n",
-    "chmod u+x hello.py\n",
-    "./hello.py\n",
+    "$ ./hello.py\n",
     "```"
    ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If a script needs to be run by a specific version of python, and not the one normally found when used on the command line, you can specify the full path to that python executable:\n",
+    "\n",
+    "```\n",
+    "#!/Users/myusername/anaconda/envs/base-py2/bin/python\n",
+    "```\n",
+    "\n",
+    "To learn more about this Unix magic, lookup [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)"
```

Comment:
I think you need a second parenthesis here - the link goes to an error page because it thinks it's ``https://en.wikipedia.org/wiki/Shebang_(Unix`` rather than ``https://en.wikipedia.org/wiki/Shebang_(Unix)``